.github/workflows/dockerbuild.yml

```diff
@@ -0,0 +1,32 @@
+name: CD-Build
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/dockerbuild.yml'
+      - 'website/**'
```

Comment:
So this will run only on merges and not on Pull Requests. We should be building on every PR and pushing when a PR gets merged. Is it correct understanding @ksatchit ?

Reply:
Yes

Reply:
@Jonsy13 We need to have both events here `Pull_Request` and `Push` and docker push should only work when a PR gets merged.

One such example is here : https://github.com/litmuschaos/litmus-e2e/blob/7550bbad4064654f303120dffdc01ac4c1a07d46/.github/workflows/build.yml

The other way is we can create a separate files for PR and push events(say `build.yml` and `push.yml` where `build.yml` should not contain docker push command).

Reply:
Building on PR is done via ci job in `main.yml`. Resolving this.